qa/tasks/userdata_setup.yaml

```diff
@@ -14,6 +14,8 @@
   #!/usr/bin/env bash
 
   # mount a NFS share for storing logs
+  sed -i 's/archive.ubuntu.com/old-releases.ubuntu.com/' /etc/apt/sources.list
```

Comment:
[off topic] but, why are we still testing on ubuntu precise?

Reply:
This task wants to boot a VM and uses an old qcow2 image as a base.  Replacing that image with something newer would be better, but I don't know what tweaks were applied at the time it was generated so it might take a bit of time to debug.